lms/envs/test.py

```diff
@@ -619,3 +619,47 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
```

Comment:
It seems like most of the settings in this file aren't necessary or related to the added components. Should we remove this part of the commit?

Reply:
Alright @kaizoku I've removed the changes that were included from upstream. The settings bellow are present in the upstream PR. What do you think?